src/DumpCommand.php

```diff
@@ -31,6 +31,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $shell = $input->getOption('shell');
         $script = $input->getArgument('script');
 
+        if (PHP_OS === 'WINNT') {
+            $shell = preg_replace('/\.exe$/i', '', $shell);
```

Comment:
How about using `basename`? Regex sounds overkill for removing the trailing suffix.

```suggestion
            $shell = basename($shell, '.exe');
```

Reply:
@bamarni I wonder would you mind doing it in https://github.com/bamarni/symfony-console-autocomplete/blob/53ecb96122df862963f69551ea801a3482df9250/src/DumpCommand.php#L23. Assuming non-Windows OS won't have a trailing `.exe` extension name.

Reply:
Yeah sounds good :+1:

Reply:
@bamarni done.